lib/resources/Order.js

```diff
@@ -0,0 +1,151 @@
+const Resource = require('../Resource')
+
```

Comment:
In order to fix json order test "should return something" i think here can be 

`const { API_BASE } = require('../constants')`

as API_BASE is not sharing his value from Resource import